src/accordionpane/example/index.ts

```diff
@@ -15,8 +15,9 @@ export class App extends WidgetBase<WidgetProperties> {
 	private _openKeys = new Set<string>();
 	private _theme: {};
 
-	themeChange(event: TypedTargetEvent<HTMLInputElement>) {
```

Comment:
Why was this changed?

Reply:
I mean I understand it's the function covariance stuff but I'm curious if this is our only option.

Reply:
At this time, yes, it's our only option. @agubler and @matt-gadd talked about changing to non-bubbling events in `widget-core/vdom` (which would mean we could rely on the type of the target), so I wasn't willing to go in and update those signatures to get this type of thing to work.